explainer.md

```diff
@@ -1,22 +1,26 @@
 ## Explainer
 
-Author: Eric Willigers &lt;<ericwilligers@chromium.org>&gt;<br>
-Author: Jay Harris &lt;<harrisjay@chromium.org>&gt;<br>
-Author: Raymes Khoury &lt;<raymes@chromium.org>&gt;
+Authors: 
+* Eric Willigers &lt;<ericwilligers@chromium.org>&gt;<br>
+* Jay Harris &lt;<harrisjay@chromium.org>&gt;<br>
+* Darwin Huang &lt;<huangdarwin@chromium.org>&gt;<br>
+* Raymes Khoury &lt;<raymes@chromium.org>&gt;
 
 ### Motivation
 
-This proposal gives web applications a way to register their ability to handle (read, stream, edit) files with given MIME types and/or file extensions.
+This proposal gives web applications a way to register their ability to handle (read, stream, edit) files with given MIME types and/or file extensions. This then allows an operating system's file manager or other native UX to select a PWA to handle the file.
 
-This has many use cases. An image editor might be able to display and edit a number of image formats. A movie player might understand a video format. A data visualizer might generate graphs from .CSV files. A Logo environment might interactively step through Logo programs.
+This has many use cases. For example:
+* An image editor could display and edit a number of image formats.
+* A movie player could play a video format.
+* A Javascript environment could interactively step through Javascript programs.
+* A data visualizer could generate graphs from .CSV files.
 
-Some more use cases and motivations are listed in the [Ballista explainer](https://github.com/chromium/ballista/blob/master/docs/explainer.md).
```

Comment:
Why not mine this explainer for a couple more use cases that are relevant before removing the reference?

Reply:
I didn't completely remove the reference. I've added a references section at the bottom of the explainer, which still has this link.

The File Handling proposal is a subset of the Ballista proposal, so I think the existing use cases section, already has a good amount of use cases, at 4. I think adding more use cases could be spammy without adding much value.

I considered replacing the image editor example with a document editor example, since we have [a demo](https://flawless-pitch-conifer.glitch.me/) that can roughly do this already, and it should be a lot easier for developers to easily envision creating a demo for this use case, but the explainer's example use case involves image editing, so I chose to leave this as is. WDYT?

Reply:
I'm fine with either approach.  The document editor example might be nice, though, since as you point out it's a live example.

Reply:
Sounds good. I've replaced the image editor example with a document editor example. I won't link the demo for now since the API may still change, but we certainly could link it later.